client/pages/car-details.jsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { Card, Table } from 'react-bootstrap';
+
+class CarDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      carTitle: '',
+      carPhoto: '',
+      carRecords: []
+    };
+    this.makeTable = this.makeTable.bind(this);
+  }
+
+  componentDidMount() {
+    fetch(`/api/garage/details/${this.props.vehicleId}`)
+      .then(result => result.json())
+      .then(result => {
+        const { year, make, model, photoUrl } = result[0];
+        this.setState({
+          carTitle: `${year} ${make} ${model}`,
+          carPhoto: photoUrl
+        });
+      });
+    fetch(`/api/garage/recent-history/${this.props.vehicleId}`)
+      .then(result => result.json())
+      .then(result => {
+        this.setState({
+          carRecords: result[0].records
+        });
+      })
+      .catch(err => console.error(err));
+  }
+
+  makeDescription(records) {
+    const newArr = [];
+    let newObj = {
+      datePerformed: records[0].datePerformed,
+      maintenanceName: records[0].maintenanceName,
+      mileage: records[0].mileage
+    };
```

Comment:
I had to think really hard about wth this method was doing. Try renaming it to `combineSameDayRecords`.

Reply:
Renamed function to combineSameDayRecords
![image](https://user-images.githubusercontent.com/48267398/163467142-27ebdceb-ad70-49f0-a63a-9ead0db4c891.png)